lib/ansible/modules/cloud/memset/memset_zone.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Simon Weald <ansible@simonweald.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: memset_zone
+author: "Simon Weald (@analbeard)"
+version_added: "2.6"
+short_description: Manage zones.
```

Comment:
This should include company & product name so it it clear on http://docs.ansible.com/ansible/devel/modules/list_of_cloud_modules.html#scaleway